tap_slack/streams.py

```diff
@@ -32,9 +32,26 @@ def get_url_params(self, context, next_page_token):
     def post_process(self, row, context):
         "Join the channel if not a member, but emit no data."
         row = super().post_process(row, context)
-        if not row["is_member"]:
-            self._join_channel(row["id"])
-        return row
+        # return all in selected_channels or default to all, exclude any in excluded_channels list
+        channel_name = row["id"]
```

Comment:
Variable name should probably be `channel_id`

Reply:
Good point! Updated.